aim/sdk/objects/image.py

```diff
@@ -56,6 +56,8 @@ def __init__(self, image, caption: str = '', format='png', quality=85, optimize=
             self._from_tf_tensor(image, params)
         elif inst_has_typename(image, ['numpy', 'array']):
             self._from_numpy_array(image, params)
+        elif inst_has_typename(image, ['Figure.matplotlib.figure']):
```

Comment:
@devfox-se I guess this need to be changed to `['Figure', 'matplotlib', 'figure']` based on `inst_has_typename` function implementation

Reply:
Will change it only for `Image` class